R/accordion.R

```diff
@@ -151,7 +151,7 @@ accordion_panel <- function(title, ..., value = title, icon = NULL) {
     class = "accordion-item",
     "data-value" = value,
     # TODO: can we provide a way to put more stuff in the header? Like maybe some right-aligned controls?
-    h2(class = "accordion-header", btn),
+    span(class = "accordion-header h2", btn),
```

Comment:
Is this bad for accessibility?

Reply:
I didn't notice any difference in behavior in VoiceOver, so I think this is fine (and I'll add a comment about why we're not using `<h2>`)